### DISCUSSION TITLE
Need sort-exports like sort-imports

### DISCUSSION THREAD

**Discussion by mockingjet** [eslint/eslint#16600] [Ideas] [answered]
We have `sort-imports` rules enabled in config, but lack `sort-exports` to make both consistent.

Is `sort-exports` going to be implemented in eslint core?

Or using this [plugin](https://www.npmjs.com/package/eslint-plugin-sort-exports) is fine?

**Comment by mdjermanovic** [comment, marked as answer]
> Is `sort-exports` going to be implemented in eslint core?

No. Per our rules policies, we will only accept new rules when they relate to new ECMAScript features that reached stage 4 in the preceding 12 months, so this rule doesn't qualify. Please find more details in this blog post:

https://eslint.org/blog/2020/05/changes-to-rules-policies/

> Or using this [plugin](https://www.npmjs.com/package/eslint-plugin-sort-exports) is fine?

Yes, using plugins is fine.
